core/embed/rust/src/ui/display/font.rs

```diff
@@ -85,7 +82,7 @@ impl Glyph {
         c_data >> 4
     }
 
-    pub fn bitmap(&self) -> Bitmap<'static> {
+    pub fn bitmap(&self) -> Bitmap<'a> {
```

Comment:
speaking of: it looks like `Bitmap::new` does not check that there's enough data provided here? @cepetr can you comment?